cmd/action/migrate.go

```diff
@@ -77,6 +77,15 @@ the migration directory state to the desired schema. The desired state can be an
 		Args: cobra.MaximumNArgs(1),
 		RunE: CmdMigrateDiffRun,
 	}
+	// MigrateHashCmd represents the migrate hash command.
+	MigrateHashCmd = &cobra.Command{
+		Use:   "hash",
+		Short: "Hash creates an integrity hash file for the migration directories.",
+		Long: `'atlas migrate hash' computes the integrity hash sum of the migration directory and stores it in the atlas.sum file.
+This command should be used whenever a manual change in the migration directory was made.`,
+		Example: `  atlas migrate hash --force`,
```

Comment:
I want to prevent accidental re-hashes on a "broken" migration directory. All `migrate` commands check for migration integrity before doing their stuff. 

And since `atlas migrate hash/sum` is a potentially "destructive" operation, I wanted to raise more awareness when using this command.